src/main/java/sirius/biz/translations/MultiLanguageStringExtender.java

```diff
@@ -0,0 +1,111 @@
+import sirius.db.mixing.Mapping;
+import sirius.db.mixing.Property;
+import sirius.kernel.commons.Context;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.commons.Value;
+import sirius.kernel.di.std.Register;
+
+import javax.annotation.Nullable;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+/**
+ * Handles importing and exporting for {@link MultiLanguageString}.
+ * <p>
+ * Note that the effective language to import and export can be controlled via {@link MultiLanguageStringHelper}.
+ */
+@Register
+public class MultiLanguageStringExtender implements EntityImportHandlerExtender {
+
+    @Override
+    public void collectLoaders(BaseImportHandler<? extends BaseEntity<?>> handler,
+                               EntityDescriptor descriptor,
+                               ImporterContext context,
+                               BiConsumer<Mapping, BiConsumer<Context, Object>> loaderCollector) {
+        for (Property property : descriptor.getProperties()) {
+            if (property instanceof MultiLanguageStringProperty) {
+                loaderCollector.accept(Mapping.named(property.getName()),
+                                       (data, entity) -> load((MultiLanguageStringProperty) property,
+                                                              context,
+                                                              data,
+                                                              entity));
+            }
+        }
+    }
+
+    private void load(MultiLanguageStringProperty property, ImporterContext context, Context data, Object entity) {
+        MultiLanguageString multiLanguageString = property.getMultiLanguageString(entity);
+        Value value = data.getValue(property.getName());
+
+        // MultiLanguageStringValue can be directly applied to the target string...
+        if (value.get() instanceof MultiLanguageStringHelper.MultiLanguageStringValue) {
+            MultiLanguageStringHelper.MultiLanguageStringValue multiLanguageStringValue =
+                    value.get(MultiLanguageStringHelper.MultiLanguageStringValue.class, null);
+            if (multiLanguageStringValue.replace) {
+                multiLanguageString.clear();
+            }
+            multiLanguageString.modify().putAll(multiLanguageStringValue.data);
+            return;
+        }
+
+        // Otherwise we use the value as string and applied as configured by the MultiLanguageStringHelper...
+        MultiLanguageStringHelper multiLanguageStringHelper =
+                context.getImporter().findHelper(MultiLanguageStringHelper.class);
+        if (multiLanguageStringHelper.isReplaceOnImport()) {
+            multiLanguageString.clear();
+        }
+
+        if (Strings.isFilled(multiLanguageStringHelper.getImportLanguage())) {
+            multiLanguageString.put(multiLanguageStringHelper.getImportLanguage(), value.getString());
+        } else if (multiLanguageString.isWithFallback()) {
```

Comment:
I think we must split this if. The provided text could be used both as language and fallback...
if isWithFallback -> setFallback
if getImportLanguage non empty -> put lang

The if in the export does it right!